docs/conf/portland/2020/news/cfp-reminder.rst

```diff
@@ -0,0 +1,52 @@
+:template: {{year}}/generic.html
+
+.. post:: January 7, 2020
+   :tags: portland-2020, website, cfp, tickets
+
+
+CFP Reminder, Tickets on Sale, Grant Program
+============================================
+
+Happy New Year, documentarians!
+
+As we get ready to start the year, we'd like to remind you that the `call for proposals <https://www.writethedocs.org/conf/portland/2020/cfp/>`_ will be open until **January 10th**, and to announce that the `tickets for the conference are officially on sale <https://www.writethedocs.org/conf/portland/2020/tickets/>`_. Read on for more details on both.
+
+Submit your talk by January 10th
+--------------------------------
+
+There are still a few days left to submit your proposal! Check out the `Call for Proposals page <https://www.writethedocs.org/conf/portland/2020/cfp/#submit-your-proposal>`_ for more details about what we are looking for and how to submit your proposal.
```

Comment:
```suggestion
There are still a few days left to submit your proposal! Check out the `Call for Proposals page <https://www.writethedocs.org/conf/portland/2020/cfp/#submit-your-proposal>`_ for more details about what we are looking for.
```

just cos it feels a bit repeaty otherwise